packages/plugin-cloud-storage/src/types.ts

```diff
@@ -79,7 +80,7 @@ export interface CollectionOptions {
 }
 
 export interface PluginOptions {
-  collections: Record<string, CollectionOptions>
```

Comment:
What is the reasoning for this to be a partial?

Reply:
Because if it's not partial, you have to define _all_ the collections there.
<img width="470" alt="image" src="https://github.com/user-attachments/assets/6c5d05bd-01a2-49d8-aa9b-9e31b8766f4f">